src/fides/api/ops/service/connectors/consent_email_connector.py

```diff
@@ -84,6 +94,34 @@ def test_connection(self) -> Optional[ConnectionTestStatus]:
                             Consent(data_use="advertising", opt_in=False),
                             Consent(data_use="improve", opt_in=True),
                         ],
+                        privacy_preferences=[
+                            MinimalPrivacyPreferenceHistorySchema(
+                                preference=UserConsentPreference.opt_in,
+                                privacy_notice_history=PrivacyNoticeHistorySchema(
+                                    name="Targeted Advertising",
+                                    regions=["us_ca"],
+                                    id="test_1",
+                                    privacy_notice_id="12345",
+                                    consent_mechanism=ConsentMechanism.opt_in,
+                                    data_uses=["advertising.first_party.personalized"],
+                                    enforcement_level=EnforcementLevel.system_wide,
+                                    version=1.0,
+                                ),
+                            ),
+                            MinimalPrivacyPreferenceHistorySchema(
+                                preference=UserConsentPreference.opt_out,
+                                privacy_notice_history=PrivacyNoticeHistorySchema(
+                                    name="Analytics",
+                                    regions=["us_ca"],
+                                    id="test_2",
+                                    privacy_notice_id="67890",
+                                    consent_mechanism=ConsentMechanism.opt_out,
+                                    data_uses=["improve.system"],
+                                    enforcement_level=EnforcementLevel.system_wide,
+                                    version=1.0,
+                                ),
+                            ),
+                        ],
```

Comment:
This is the code that sends a test email for a saas email connector.  I'm putting in new-style privacy preferences into the email as well as leaving old style consent preferences.

Reply:
good idea!  I don't think I need to mark all of them, but this is one that could be easily forgotten I think.